common/presenters/message-banner/move-to-message-banner-component.js

```diff
@@ -23,6 +24,10 @@ function moveToMessageBannerComponent({ move = {}, moveUrl, canAccess } = {}) {
     return assessmentToStartBanner({ baseUrl, canAccess, context })
   }
 
+  if (assessment?.status === 'confirmed' && assessment.handover_occurred_at) {
```

Comment:
We have tests for both of these conditions, it might be useful to have tests for each one individually, to ensure they don't use `assessmentToHandedOverBanner`

Reply:
There's already an [existing test](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1319/files#diff-24aa969c052a068ce604e9ad1063949a2d4449b4b37f7de4b3720c6239019d88R120) to ensure that without `handover_occurred_at` that the correct banner is returned, in that case `assessmentToConfirmedBanner `.

Is that what you mean?

Reply:
Oh, cool - that'll do for this PR I think :)